RESideMenu/RESideMenuController.m

```diff
@@ -412,29 +495,45 @@ - (void)showRightMenuViewController
         [self.rightMenuViewController endAppearanceTransition];
         [self.contentViewController endAppearanceTransition];
     }];
-    
+
+    // Perspective rotation animation
+    CABasicAnimation *animation = [CABasicAnimation animationWithKeyPath:@"transform.rotation.y"];
```

Comment:
Worth extracting these animation creations out to a method? These two share a lot in common with the two above.